bindings/IVector_fromTo_matlab.i

```diff
@@ -0,0 +1,55 @@
+%extend std::vector<int> {
+    // Convert to a Matlab vector
+    mxArray * toMatlab() const
+    {
+        // create Matlab vector and map it to pointer 'd'
+        size_t selfDim = self->size();
+        mxArray *p  = mxCreateDoubleMatrix(selfDim, 1, mxREAL);
+        double* d = static_cast<double*>(mxGetData(p));
+
+        // copy items from 'self' to 'd'
+        const int* selfData = self->data();
+        for(size_t i=0; i<selfDim; i++)
+        {
+            d[i] = selfData[i];
```

Comment:
Nitpick: just for clarity it would be good to have a static_cast<double>() here.